pkg/soc/soc.go

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// Package soc provides the single-owner chunk implemenation
+// and validator.
```

Comment:
suggestion to add: An soc is a chunks whose address is derived of (...)

Reply:
prefer not to get technical in the godoc